app/Http/Controllers/Consumables/ConsumableCheckoutController.php

```diff
@@ -69,6 +70,12 @@ public function store(Request $request, $consumableId)
             'note' => $request->input('note'),
         ]);
 
+        $checkedout= DB::table('consumables_users')->where('consumable_id', '=', $consumable->id)->count();
+        $available= new Consumable();
+
+        if($checkedout >= $available->numRemaining()){
```

Comment:
Couldn't we just check if `numRemaining` is greater than 0?

Reply:
life is simple, and here I am trying to make it harder. fixed! 😅